app/src/main/res/layout/bsd_channeldetail.xml

```diff
@@ -228,4 +297,43 @@
         android:layout_height="wrap_content"
         android:orientation="vertical"
         app:layout_constraintGuide_percent="0.50" />
+
+    <Button
+        android:id="@+id/channelCloseButton"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginEnd="8dp"
+        android:background="@drawable/bg_clickable_item"
+        android:textColor="@color/lightningOrange"
+        android:visibility="gone"
```

Comment:
Maybe add android:textAllCaps="false"
We do not have AllCaps buttons anywhere else (except the setup wallet button which has a completely different style anyway)